{{cookiecutter.project_shortname}}/setup.py

```diff
@@ -38,16 +38,33 @@
         ],
         'invenio_base.apps': [
             'flask_debugtoolbar = flask_debugtoolbar:DebugToolbarExtension',
+            {%- if cookiecutter.data_model == 'Custom (as part of the instance)' %}
+            '{{ cookiecutter.package_name }}_records = {{ cookiecutter.package_name }}.records:{{ cookiecutter.extension_class }}',
```

Comment:
On second thought we should probably parameterize the name of the datamodel as well, and default it to ``records``.

Reply:
@lnielsen I just started renaming, and I can see that it is going to change in many places, to keep it consistent, like urls, mappings names, schemas names, etc. I think in this case it will loose the basic invenio 'record' identity as a starting point - and for a new user it might be confusing as probably everywhere we refer to 'records'. WDYT, should I proceed?